answers.txt

```diff
@@ -236,3 +236,15 @@
 "Ignoring native features and using another product instead"
 "Disabling SE Linux to make it work"
 "Disabling AppArmor because it makes things unstable"
+"Whatever the hell an Istio might be"
+"An AWS service with more syllables than engineers"
+"Your résumé"
+"Copying and pasting from Stack Overflow"
+"A Short Stack Developer"
+"A technical preview that only works on Kelsey Hightower's laptop"
+"An endless field of service meesh"
```

Comment:
"meesh". typo? intentional?

Reply:
As gese is to geese, mesh is to meesh.

Reply:
You mean a Maish :) ?

Reply:
Very intentional. See https://twitter.com/quinnypig/status/1174233112634699776?s=21